src/Sarif.Viewer.VisualStudio/Sarif/CodeFlowLocationExtensions.cs

```diff
@@ -19,11 +19,14 @@ public static Microsoft.Sarif.Viewer.Models.CodeFlowLocationModel ToAnnotatedCod
                 {
                     model.Region = codeFlowLocation.Location.PhysicalLocation.Region;
 
-                    Uri uri = codeFlowLocation.Location.PhysicalLocation.Uri;
-                    if (uri != null)
+                    if (codeFlowLocation.Location.PhysicalLocation.FileLocation != null)
                     {
-                        model.FilePath = uri.ToPath();
-                        model.UriBaseId = codeFlowLocation.Location.PhysicalLocation.UriBaseId;
+                        Uri uri = codeFlowLocation.Location.PhysicalLocation.FileLocation.Uri;
```

Comment:
As in the previous comment, you can collapse this to a single test with `FileLocation?.Uri`.

Reply:
Fixed